ee/clickhouse/migrations/0029_cohortpeople_version.py

```diff
@@ -0,0 +1,20 @@
+from infi.clickhouse_orm import migrations
+
+from posthog.client import sync_execute
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+ADD_COLUMNS_BASE_SQL = """
+ALTER TABLE {table}
+ON CLUSTER '{cluster}'
+ADD COLUMN IF NOT EXISTS version UInt64,
+MODIFY ORDER BY (team_id, cohort_id, person_id, version)
```

Comment:
This seems safe according to [docs](https://clickhouse.com/docs/en/sql-reference/statements/alter/order-by/)

Reply:
I agree that those docs make this seem safe, but it still makes me nervous 😬. Would definitely want someone from infra to give a review here before we run this. @fuziontech @hazzadous

Reply:
Looks fine to me. But I guess the IF NOT EXISTS doesn't make sense as in the case that it exists, the modify order by would fail.

Reply:
Why would the modify fail if version already exists?

Reply:
According to the docs, the additional columns for order by need to not already exist. I don't think it really matters much though, it will just be a different error message I presume.

Reply:
Ahh i see. Hmm let me see if there's a smoother way

Reply:
Gave it a test and it seems even if the statement is called when the column already exists, nothing happens, it just returns OK